packages/cactus-plugin-ledger-connector-xdai/src/main/typescript/web-services/deploy-contract-solidity-bytecode-endpoint.ts

```diff
@@ -91,12 +92,21 @@ export class DeployContractSolidityBytecodeEndpoint
     try {
       const resBody = await this.options.connector.deployContract(reqBody);
       res.json(resBody);
-    } catch (ex) {
+    } catch (ex: unknown) {
+      const stack = LogHelper.getExceptionStack(ex);
+      const messages = LogHelper.getExceptionMessage(ex);
       this.log.error(`Crash while serving ${reqTag}`, ex);
-      res.status(500).json({
-        message: "Internal Server Error",
-        error: ex?.stack || ex?.message,
-      });
+      if (ex instanceof Error) {
```

Comment:
@Leeyoungone Let's wait with this until https://github.com/hyperledger/cactus/issues/1381 is resolved as well because that's what it really depends on and I don't know for sure yet how that's going to play out.

Reply:
Sounds good! So sorry, I've just been plugging away at this epic -- I can go ahead and wait till the dependent PRs are merged into main. Feel free to ignore the current PRs I have open for endpoint error handling !